specification/mcpp/resource-manager/Microsoft.CloudPartnerProgramMembership/preview/2023-02-09-preview/examples/getMcppPurchaseDetails.json

```diff
@@ -0,0 +1,25 @@
+{
+    "parameters": {
+      "resourceGroupName": "McppPurchases",
+      "api-version": "2023-02-09-preview",
+      "subscriptionId": "291bba3f-e0a5-47bc-a099-3bdcb2a50a05",
+      "resourceName": "microsoftactionpack"
+    },
+    "responses": {
+      "200": {
+        "body": {
+          "id": "microsoftactionpack",
+          "name": "microsoftactionpack",
+          "type": "string",
+          "properties": { 
+            "orderNo": "2302319360",
```

Comment:
Response should have productCode inside properties.

Reply:
ack, will make this change.